install_protoc_osx_x86_64.sh

```diff
@@ -0,0 +1,10 @@
+# http://google.github.io/proto-lens/installing-protoc.html
+PROTOC_ZIP=protoc-3.12.0-osx-x86_64.zip
+curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.12.0/$PROTOC_ZIP
+sudo unzip -o $PROTOC_ZIP -d /usr/local bin/protoc
+sudo unzip -o $PROTOC_ZIP -d /usr/local 'include/*'
+rm -f $PROTOC_ZIP
+
+# https://grpc.io/docs/languages/go/quickstart/
+GO111MODULE=on go get -v -u google.golang.org/protobuf/cmd/protoc-gen-go@v1.25
+GO111MODULE=on go get -v -u google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1
```

Comment:
@nicovogelaar Thank you for this effort, but we would rather not  include shell scripts for installing protoc like this on OSX.

We should have a platform agnostic way of installing all the dependencies for local builds (and for CI builds).

E.g. a Docker image for local, and possibly to reuse it for the CI.

Reply:
Sure, I will remove it. It may be good to document it somewhere to make it easier for other developers to contribute. Keep in mind that different versions of protoc will have different outputs of generated code.